agent.go

```diff
@@ -593,6 +599,16 @@ func (p *pod) listenToUdevEvents(done chan struct{}) {
 	}()
 }
 
+func (p *pod) onlineVCPU(syspath string) {
+	onlinePath := filepath.Join(syspath, "online")
```

Comment:
I assume the syspath you're gonna get here is going to be something like:
```
/sys/devices/system/cpu/cpuN
```
Right ?
Could be nice to check the path is the expected pattern before to concatenate it with `"online"`.

Reply:
I'd prefer to just check if the file exist

Reply:
Sounds good.